bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -196,6 +200,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         return;
                     }
                 }
+            case RenaultBindingConstants.CHANNEL_UPDATE_NOW:
```

Comment:
Why not implement `RefreshType` instead?

Reply:
@jlaur Thanks for the feedback.  Sorry it is hard for me to find time at the moment to process it.  I did however investigate the RefreshType.  It seams to me that this is used by the UI to refresh the state of items etc... Is it possible to trigger a binding refresh from a rule?  The problem we wish to solve is described here: https://community.openhab.org/t/betatest-renault-ze-services-binding/72226/222?u=doug_culnane

"Now that we have 2 cars and only one wallbox, I’d like to know which of the car is plugged to the charger.
With the automated polling I will receive this information after at least 5 minutes (in my case), that could be to long.

I have a workaround for this, but its also a little complicated.
I disable the car Thing with an HttpPutRequest - wait for 1 second - and enable the car thing again with the HttpPutRequest."

The "updatenow" channel is a better work around that this.  However maybe there is a Item.refresh() function that I could implement on the channel and this would work better with the openHAB platform?

Reply:
> Is it possible to trigger a binding refresh from a rule?

Yes, for example in a DSL rule:
```java
sendCommand(itemName, "REFRESH")
```